### ISSUE TITLE
Navbarcolor ('Dark navbar' mode): Switch color does not change when activated

### ISSUE THREAD

**Issue by tmtsc** [moodle-an-hochschulen/moodle-theme_boost_union#686]
Regarding the setting "navbarcolor" - When using the ‘Dark navbar with light font color’ mode, the edit switch does not change color after activation and remains white, which can be confusing (only the positioning changes from left to right).
![navbar-dark-switch](https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/assets/105368391/a93259b1-5500-464d-bacb-4e710524b535)

The expected behavior is that it should be displayed with a blue fill (or a different color) similar to the ‘Light navbar’ mode, in order to stand out from the inactive state.
"Light Navbar" mode:
![navbar-light-switch2](https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/assets/105368391/d930c8b0-8593-4cc3-90bd-495dbfb59d3d)

Tested on Firefox and Chrome (Windows and Android).

**Comment by MalteHellwig** [comment]
I confirm this kind of behaviour. I would love that feature for the dark nav bar as well. 

Greets from Wilhelmshaven!

**Comment by anna-sparkfore** [comment]
Agree! And the same if you select the option of using the Brand colour on the navbar:
![image](https://github.com/user-attachments/assets/2d877b87-be3f-47c8-879c-e3960cf1ee49)

**Comment by acquaalta** [comment]
Hi,
+1 for this bug report or feature request. 
When using the dark navbar, it's difficult to visually identify if the edit mode is enabled or disabled.

**Comment by anna-sparkfore** [comment]
The custom CSS does not work too well with our brand colour unfortunately:

<img width="277" height="67" alt="Image" src="https://github.com/user-attachments/assets/fe2850ec-c016-4413-844f-dc23502fa0dd" />
